mmv1/products/integrationconnectors/ManagedZone.yaml

```diff
@@ -0,0 +1,109 @@
+# Copyright 2023 Google Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Api::Resource
+name: "ManagedZone"
+description: |
+  An Integration connectors Managed Zone.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    "Official Documentation": "https://cloud.google.com/integration-connectors/docs"
+  api: "https://cloud.google.com/integration-connectors/docs/reference/rest/v1/projects.locations.global.managedZones"
+base_url: projects/{{project}}/locations/global/managedZones
+self_link: projects/{{project}}/locations/global/managedZones/{{name}}
+timeouts: !ruby/object:Api::Timeouts
+  insert_minutes: 5
+  update_minutes: 5
+  delete_minutes: 5
+create_url: projects/{{project}}/locations/global/managedZones?managedZoneId={{name}}
+update_verb: :PATCH
+update_mask: true
+autogen_async: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "integration_connectors_managed_zone"
+    primary_resource_id: "testmanagedzone"
+    test_env_vars:
+      org_id: :ORG_ID
+      billing_account: :BILLING_ACCT
+    vars:
+      managed_zone_name: "test"
+      # Resource creation race
+    skip_vcr: true
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: "name"
+    base_url: "{{op_id}}"
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 5
+      update_minutes: 5
+      delete_minutes: 5
+  result: !ruby/object:Api::OpAsync::Result
+    path: "response"
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: "done"
+    complete: true
+    allowed:
+      - "true"
+      - "false"
+  error: !ruby/object:Api::OpAsync::Error
+    path: "error"
+    message: "message"
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: "name"
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      Name of Managed Zone needs to be created.
+
+properties:
+  - !ruby/object:Api::Type::Time
```

Comment:
We usually omit create time and update time fields from new resources. Is there a reason to include them here?